project-minimization-core/src/main/kotlin/org/plan/research/minimization/core/algorithm/dd/DDAlgorithm.kt

```diff
@@ -2,7 +2,7 @@ package org.plan.research.minimization.core.algorithm.dd
 
 import org.plan.research.minimization.core.model.*
 
-typealias DDAlgorithmResult<T> = List<T>
+data class DDAlgorithmResult<T>(val survived: List<T>, val deleted: List<T>)
```

Comment:
Survived sounds a bit... harsh? I think it could be named smth like `left` or `reproducingSet`, WDYT?